functions/src/winloss.ts

```diff
@@ -0,0 +1,27 @@
+import * as functions from "firebase-functions";
+import { lobbyCollection } from "./firestore-collections";
+import { isLobbyRequest } from "./firestore-functions-types";
+import { Lobby } from "./firestore-types/lobby";
+
+// this function will likely be obsolete once timers are introduced
+export const lobbyReturn = functions.https.onCall(async (data: unknown, context) => {
```

Comment:
lets put an explict return type here like 191b0bf80c59858ed7b9e1b52d2e04ce9b9f2f9f

Reply:
I commented in line 6 that this function probably won't be used in the future, do you want me to just remove it completely now or when the timers are actually implemented and fix it now?

Reply:
that depends on how close timers are to being done

Reply:
For now I'll just keep the function updated with the new changes